backend/src/converter/endpoints.ts

```diff
@@ -1,5 +1,5 @@
 import { validateConverter } from "./service";
-import { getCollections } from "../db";
+import { getCollections } from "../service";
```

Comment:
Średnio mi się podoba ta zmiana nazwy pliku. Service jest imo zbyt ogólnie. Jak już to chyba bym preferował nazwę `dbService` albo ewentualnie utworzyć nowy folder `db` (lub `database`), a w nim plik `service` ale to też nie wiem czy nie jest nadmiarowe, bo nie przewiduję, żeby coś jeszcze oprócz tego pliku mogłoby się znaleźć w tym folderze

Reply:
No właśnie dodałem tam funkcję która nie jest związana z db i dlatego zmieniłem nazwę. Obecnie nie ma lepszego miejsca na `asyncTryCatchAssign`